### PR TITLE
Updates README

### PR DESCRIPTION
**Pull request by sarony**
- This is for trello card --> https://trello.com/c/ygqIHbiH/20-hangman-wireframe-mockup
- I'm unsure about the copy. Please feel free to make suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
 Trello link here: https://trello.com/b/ffCjlMki/js-tuesday
 
-REBASE ME!
-
```
Comment (jamalhansen) [inline README.md:4]: Great new content!  I liked the All your rebase tho :)

**Comment by sarony** [comment]
@sarony This looks good to me! Feel free to merge :)

File: README.md [hunk 2]
```diff
@@ -1,10 +1,7 @@
 Trello link here: https://trello.com/b/ffCjlMki/js-tuesday
```
Comment (sarony) [inline README.md]: I don't believe this is the correct trello link. Did you mean to use a link from ruby-monday?

**Comment by jamalhansen** [comment]
Thanks for updating, that looks great!  Happy merging!

**Pull request merged**